discovery/kv/kv.go

```diff
@@ -61,7 +61,15 @@ func (s *Discovery) Initialize(uris string, heartbeat time.Duration, ttl time.Du
 			EphemeralTTL: s.ttl,
 		},
 	)
-	return err
+
+	if err != nil {
+		if err == store.ErrInvalidTTL {
```

Comment:
Is it really necessary to check this?

AFAIK `CreateStore` never fails unless:
- The backend doesn't exist
- The TTL is invalid

Both should just fail